src/sections/channels.tsx

```diff
@@ -0,0 +1,492 @@
+              <TableCell>status</TableCell>
+              <TableCell>dedicated funds</TableCell>
+              <TableCell>actions</TableCell>
+            </TableRow>
+          </TableHead>
+          {tabIndex === 0 && (
+            <TableBody>
+              {Object.entries(channels?.incoming ?? []).map(
+                ([, channel], key) => (
+                  <TableRow key={key}>
+                    <TableCell component="th" scope="row">
+                      {key}
+                    </TableCell>
+                    <TableCell>{getAliasByPeerId(channel.peerId)}</TableCell>
+                    <TableCell>{channel.status}</TableCell>
+                    <TableCell>{channel.balance}</TableCell>
+                    <TableCell>
+                      <button
+                        onClick={() =>
+                          handleCloseChannels(
+                            'incoming',
+                            channel.peerId,
+                            channel.channelId
+                          )
+                        }
+                      >
+                        Close
+                      </button>
+                      {closingStates[channel.channelId]?.closing && (
+                        <CircularProgress />
+                      )}
+                      {closingStates[channel.channelId]?.closeSuccess && (
+                        <div>Close Success</div>
+                      )}
+                      {closingStates[channel.channelId]?.closeErrors.map(
+                        (error, index) => (
+                          <div key={index}>{error.error}</div>
+                        )
+                      )}
+                      <hr />
+                      {fundChannel(channel.channelId, channel.peerId)}
+                      {fundingStates[channel.channelId]?.funding && (
+                        <CircularProgress />
+                      )}
+                      {fundingStates[channel.channelId]?.fundingSuccess && (
+                        <div>Funding Success</div>
+                      )}
+                      {fundingStates[channel.channelId]?.fundingErrors.map(
+                        (error, index) => (
+                          <div key={index}>{error.error}</div>
+                        )
+                      )}
+                    </TableCell>
+                  </TableRow>
+                )
+              )}
+            </TableBody>
+          )}
+          {tabIndex === 1 && (
+            <TableBody>
+              {Object.entries(channels?.outgoing ?? []).map(
+                ([, channel], key) => (
+                  <TableRow key={key}>
+                    <TableCell component="th" scope="row">
+                      {key}
+                    </TableCell>
+                    <TableCell>{getAliasByPeerId(channel.peerId)}</TableCell>
+                    <TableCell>{channel.status}</TableCell>
+                    <TableCell>{channel.balance}</TableCell>
```

Comment:
We need a function in utils that will format balances. Something like format `formatEther(balance)`, as we do not have `500000000000000000` but  `0.5 mHOPR`. Also add `mHOPR` at the end here.